tests/test_raster_area_stats_univar.py

```diff
@@ -1,117 +1,188 @@
 # -*- coding: utf-8 -*-
 import unittest
-import time
 from pprint import pprint
 from flask.json import loads as json_load
 from flask.json import dumps as json_dump
+
 try:
     from .test_resource_base import ActiniaResourceTestCaseBase, URL_PREFIX
-except:
+except Exception:
     from test_resource_base import ActiniaResourceTestCaseBase, URL_PREFIX
 
 
 __license__ = "GPLv3"
-__author__     = "Sören Gebbert"
-__copyright__  = "Copyright 2016, Sören Gebbert"
+__author__ = "Sören Gebbert"
+__copyright__ = "Copyright 2016, Sören Gebbert"
 __maintainer__ = "Soeren Gebbert"
-__email__      = "soerengebbert@googlemail.com"
+__email__ = "soerengebbert@googlemail.com"
 
 
 JSON = {
-"type": "FeatureCollection",
-"crs": { "type": "name", "properties": { "name": "urn:ogc:def:crs:EPSG::3358" } },
-"features": [
-{ "type": "Feature", "properties": { "fid": "swwake_10m.0" }, "geometry": { "type": "Polygon", "coordinates": [ [ [ 630000.0, 215000.0 ], [ 630000.0, 228500.0 ], [ 645000.0, 228500.0 ], [ 645000.0, 215000.0 ], [ 630000.0, 215000.0 ] ] ] } }
-]
+    "type": "FeatureCollection",
+    "crs": {
+        "type": "name",
+        "properties": {"name": "urn:ogc:def:crs:EPSG::3358"},
+    },
+    "features": [
+        {
+            "type": "Feature",
+            "properties": {"fid": "swwake_10m.0"},
+            "geometry": {
+                "type": "Polygon",
+                "coordinates": [
+                    [
+                        [630000.0, 215000.0],
+                        [630000.0, 228500.0],
+                        [645000.0, 228500.0],
+                        [645000.0, 215000.0],
+                        [630000.0, 215000.0],
+                    ]
+                ],
+            },
+        }
+    ],
 }
 
 
 class RasterAreaStatsTestCase(ActiniaResourceTestCaseBase):
+    """
+    def test_async_raster_area_stats_json(self):
+        rv = self.server.post(
+            URL_PREFIX
+            + "/locations/nc_spm_08/mapsets/PERMANENT/raster_layers/elevation/"
+            "area_stats_univar_async",
+            headers=self.admin_auth_header,
+            data=json_dump(JSON),
+            content_type="application/json",
+        )
+
+        import pdb
+
+        pdb.set_trace()
```

Comment:
Why is `pdb` imported and called here various times?

Reply:
I think there I tried some time ago to fix the tests. They will be removed when I fix the tests in another MR